test/GovPool.test.js

```diff
@@ -78,6 +80,10 @@ describe("GovPool", () => {
 
   const getProposalByIndex = async (index) => (await govPool.getProposals(index - 1, 1))[0].proposal;
 
+  function toPercent(num) {
```

Comment:
You can add this function to `scripts/utils/utils.js` file

Reply:
done